metaflow/datatools/s3op.py

```diff
@@ -270,21 +414,29 @@ def verify_results(urls, verbose=False):
             exit(ERROR_VERIFY_FAILED, url)
         if expected != got:
             exit(ERROR_VERIFY_FAILED, url)
-
-def generate_local_path(url):
+        if url.content_type or url.metadata:
```

Comment:
Minor nit - L:408 we should print the actual url. On the console - I get `verifying <__main__.S3Url object at 0x7fb5688b0b38>, expected 1528`

Reply:
Will fix. Good catch.